app/tests/content/test_user_integration.py

```diff
@@ -469,3 +477,26 @@ def test_destroy_other_user_as_index_user(member, user, api_client):
     response = client.delete(url)
 
     assert response.status_code == status.HTTP_200_OK
+
+
+@pytest.mark.django_db
+def test_expired_user_event_shows_as_expired(member, api_client):
+    """ "All the events listed as expired should be expired"""
+    client = api_client(user=member)
+
+    two_days_ago = timezone.now() - timedelta(days=2)
+    event = EventFactory(end_date=two_days_ago)
+
+    registration = RegistrationFactory(user=member, event=event)
+
+    url = _get_user_events_url()
+
+    query_params = {"expired": "true"}
+
+    response = client.get(url, data=query_params)
+
+    assert response.status_code == status.HTTP_200_OK
+
+    registrations = response.json().get("results")
+    for registration in registrations:
+        assert registration.get("expired")
```

Comment:
Dette er en bra test!

Som dere kanskje ser så finnes det ikke noen andre tester for disse ekstra funksjonene (blame de som lagde funksjonene).

Dere må dermed også lage en test som sjekker hvis dere ikke sender med expired som et param eller at den er satt til False. For da skal man få tilbake en liste med events som ikke er expired, slik som det er nå.